core/src/main/resources/db/changelog/changesets/2024_02_09_answer_viewed_language.yaml

```
@@ -0,0 +1,9 @@
+databaseChangeLog:
+  - changeSet:
+      id: add_answer_viewed_lang
+      author: mbemis
+      changes:
+        - addColumn:
+            tableName: answer
+            columns:
+              - column: { name: viewed_language, type: text, constraints: { nullable: true } }
```

I think these are easier to read if we don't specify nullable:false -- that's the default
```suggestion
              - column: { name: viewed_language, type: text }
```